annotations/utils.py

```diff
@@ -37,6 +38,170 @@ def basepath(request):
     return scheme + request.get_host() + settings.SUBPATH
 
 
+def build_concept_node(concept, user):
+    """
+    Helper function to build a concept node dictionary.
+    """
+    return {
+        "label": concept.label or "",
+        "metadata": {
+            "type": "appellation_event",
+            "interpretation": concept.uri,
+            "termParts": [
+                {
+                    "position": 1,
+                    "expression": concept.label or "",
+                    "normalization": "",
+                    "formattedPointer": "",
+                    "format": ""
+                }
+            ]
+        },
+        "context": {
+            "creator": user.username,
+            "creationTime": timezone.now().strftime('%Y-%m-%d'),
+            "creationPlace": "phoenix",
+            "sourceUri": concept.authority if hasattr(concept, 'authority') else ""
```

Comment:
this needs to be the the uri of the text that has been annotated, so for now let's use the the url of the annotated text

Reply:
the above comments apply to all contexts